far2l/DE/far2ledit.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=far2ledit
+Comment=Editor from far2l file manager
+Exec=far2ledit %f
```

Comment:
I am not sure blindly copying the comments from another project is a good idea.

First, comment is secondary information which is not displayed by default. What is displayed is the name.

Second, some of these strings may be subject to copyright. Midnight commander is GPL-3+, while far2l is GPL-2+.

And finally, there may be some issues in your strings. For example, the string for Farsi seems to have unintended `<200c>` (literally) inside it. Also I translated `Far боловсруулах` from Mongolian back to English and it says `Far processing`, not `Far editor`.

I agree that having proper translation infrastructure for desktop files would be good, but we have none at the moment, and it would be better to start with the main desktop file, not with far2ledit one, so it is orthogonal to my changes.